src/components/sideBarComponents/SideBarLogOutBoxIcon.jsx

```diff
@@ -0,0 +1,23 @@
+import MoreVertTwoToneIcon from '@mui/icons-material/MoreVertTwoTone';
+import { useSideBar } from "../../contexts/";
+
+const SideBarLogOutBoxIcon = () => {
+
+    const { handleSideBarLogoutPopupDisplay } = useSideBar();
+
+    return (
+        <MoreVertTwoToneIcon
+            sx={{
+                fontSize: "medium",
+                position: "absolute",
+                right: "0.5rem",
+                '&:hover': {
+                    color: 'otherColors.lightPurple',
+                    cursor: 'pointer'
+                }
+            }}
+            onClick={() => handleSideBarLogoutPopupDisplay()} />
```

Comment:
Can simply write as onClick={handleSideBarLogoutPopupDisplay}
Will work in the same manner as described earlier.